### ISSUE TITLE
Error while missing label

### ISSUE THREAD

**Issue by Hiswe** [datuhealth/floating-label#1]
Hi,

Thank you for your plugin. 
I run into an issue recently. It throws this error when I initialise floating-label in a page containing some inputs without associated labels (a grid in my case):

```
Uncaught TypeError: Cannot read property 'classList' of nulladdClass 
@ floating-label:120showHideLabel 
@ floating-label:42evaluateInputs 
@ floating-label:67
```

I was wondering if you can handle this properly or (better for me) leave the possibility to pass an array of node to setup this plugin only on the necessary inputs :)

**Comment by mike-engel** [comment]
@Hiswe The first option will be faster to implement, so I'm going to do that first. After that's done and I get some more time, I'll allow for an optional array of Elements to be passed in which will be the only ones bound.

**Comment by Hiswe** [comment]
@mike-engel thank you very much :)

**Comment by mike-engel** [comment]
@Hiswe Just published version `2.0.0`, which should take care of this problem. If not, please let me know!
